### DISCUSSION TITLE
Not able to export iconize emojis while exporting to pdf.

### DISCUSSION THREAD

**Discussion by PrabinKuSabat** [l1xnan/obsidian-better-export-pdf#285] [General]
While exporting a single file form .md to pdf I'm not able to export the iconize plugin emoji's and they are comming as plain text.
(for example, :liberia: instead of 🇱🇷 .
Is there a way to import custom emoji style to the plugin and let it export those.
Or there is any other way?

**Comment by l1xnan** [comment]
Do you want to add an emoji icon to the title?

**Reply by PrabinKuSabat** [reply]
No.
Any image in the body pasted using the "Iconize" plugin is exported as plain text by the Better Export Pdf plugin.
Even the inbuild export to pdf does the same.

**Reply by l1xnan** [reply]
Is the official PDF export normal?

**Reply by PrabinKuSabat** [reply]
No even the official PDF export is not normal.
Basically iconize recognizes  a particular character(symbol) as the emoji initiator, and the following text is converted to the respective emoji. The name of the emoji is ended with the same character so that the plugin can recognize the emoji.
Now any export to pdf is recognizing only the text. I wan't the emoji's of the plugin(Iconize) to be incorporated into the export process. Btw Iconize has 914,539 downloads(as of now) in obsidian.
It will be helpful for many if this can be incorporated into the **EXPORT to pdf** function.
